pkg/users/local.js

```diff
@@ -101,14 +101,20 @@
 }
 
 async function getLogins() {
-    const lastlog = await cockpit.spawn(["/usr/bin/lastlog"], { environ: ["LC_ALL=C"] })
-            .catch(err => console.warn("Unexpected error when getting last login information", err));
-    const w = await cockpit.spawn(["/usr/bin/w", "-sh"], { environ: ["LC_ALL=C"] })
-            .catch(err => console.warn("Unexpected error when getting logged in accounts", err));
+    let lastlog = [];
+    try {
+        lastlog = await cockpit.spawn(["/usr/bin/lastlog"], { environ: ["LC_ALL=C"] });
+    } catch (err) {
+        console.warn("Unexpected error when getting last login information", err);
+    }
 
-    const currentLogins = w.split('\n').slice(0, -1).map(line => {
-        return line.split(/ +/)[0];
-    });
+    let currentLogins = [];
+    try {
+        const w = await cockpit.spawn(["/usr/bin/w", "-sh"], { environ: ["LC_ALL=C"] });
+        currentLogins = w.split('\n').slice(0, -1).map(line => line.split(/ +/)[0]);
+    } catch (err) {
+        console.warn("Unexpected error when getting logged in accounts", err);
+    }
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18198-20230118-133515-72233636-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)